docs/userguide.rst

```diff
@@ -251,13 +253,19 @@ The above example configures 'somename.eth' to resolve to the address of your pr
 Transferring a name
 -------------------
 
-You can transfer ownership of a name you own in the ENS registry to someone else using `setOwner`:
+You can transfer ownership of a name you own in the ENS registry to another trusted account using `setOwner`:
 
 ::
 
     > ens.setOwner(namehash('somename.eth'), newOwner, {from: eth.accounts[0]});
 
-Note, however, that if the name was acquired through a registrar, such as through an auction described above, this will not transfer ownership of the locked bid! It will also not perform any administrative tasks that a registrar might want to do.
+This way, the bidding/renewal account ``eth.accounts[0]`` can be kept separate from the day-to-day control account ``newOwner``.
```

Comment:
This isn't the only reason to set owners through ENS; only 2LD names under .eth have corresponding deeds. I would just leave out this paragraph entirely.